docs/guidance/volcano-integration.md

```diff
@@ -0,0 +1,67 @@
+# KubeRay integration with Volcano
+
+[Volcano](https://github.com/volcano-sh/volcano) is a batch scheduling system built on Kubernetes. It provides a suite of mechanisms (gang scheduling, job queues, fair scheduling policies) currently missing from Kubernetes that are commonly required by many classes of batch and elastic workloads. With the integration with Volcano, KubeRay cluster pods can be scheduled for better scheduling efficiency in multi-tenant environments.
+
```

Comment:
Would you mind identifying yourself as a maintainer of the Volcano integration and providing your GitHub handle?